src/frequenz/client/dispatch/types.py

```diff
@@ -260,8 +115,21 @@ class TimeIntervalFilter:
     """Filter by end_time < end_to."""
 
 
+class RunningState(Enum):
+    """The running state of a dispatch."""
+
+    RUNNING = "RUNNING"
+    """The dispatch is running."""
+
+    STOPPED = "STOPPED"
+    """The dispatch is stopped."""
+
+    DIFFERENT_TYPE = "DIFFERENT_TYPE"
+    """The dispatch is for a different type."""
```

Comment:
I guess this is OK if we are just moving things around for now, but the goal was to get rid of this, right? If we can do it right here, right now, I would be much happier :P

I mean, `RUNNING` and `STOPPED`, fair, but `DIFFERENT_TYPE` looks super weird as a state.